src/shipit_code_coverage/tests/test_artifacts.py

```diff
@@ -59,3 +61,27 @@ def add_dir(files):
 
     with pytest.raises(Exception, message='suite and chunk can\'t both have a value'):
         a.get(chunk='xpcshell-7', suite='mochitest')
+
+
+#I'm using the LINUX_TASK_ID as taskID here, but the problem is that,
+#when I do taskcluster.get_task_artifacts, none of the names has code-coverage-grcov.zip
+#or code-coverage-jsvm.zip as substring, so artifact.download() never calls
+#taskcluster.download_artifact(). Is there a specific TEST_TASK I should use?
```

Comment:
LINUX_TASK_ARTIFACTS is coming from https://github.com/mozilla-releng/services/blob/master/src/shipit_code_coverage/tests/fixtures/linux_task_artifacts.json, which only contains build artifacts (so the gcno file) but not test artifacts.

LINUX_TASK_ID is basically build_task_id here: https://github.com/mozilla-releng/services/blob/master/src/shipit_code_coverage/shipit_code_coverage/artifacts.py#L76.

Reply:
So, you should define a new LINUX_TEST_TASK_ARTIFACTS, which would be getArtifacts response for a Linux test task instead of a Linux build task.